src/components/accordion/accordion.yaml

```diff
@@ -0,0 +1,99 @@
+params:
+- name: id
+  type: string
+  required: true
+  description: Used to assign an `id` in the HTML for the accordion as a whole, and also a prefix for the `id`s of the section panels and the buttons that open them, so that those `id`s can be the target of `aria-labelledby` and `aria-control` attributes.
+- name: items
+  type: array
+  required: true
+  descripion: An array of sections within the accordion.
+  params:
+    - name: label
```

Comment:
I wonder if calling this `label` risks confusing things with form elements - what about `headerText`?

Reply:
label mirrors the Tab component at least...

https://design-system.service.gov.uk/components/tabs/

Reply:
@nickcolley Yeah, I think I borrowed the terminology from that. Happy to switch it to `heading` though. I suggest we used `heading` rather than `headingText`, as that would allow us to support `heading.text` and `heading.html` if necessary.